libfreerdp/cache/pointer.c

```diff
@@ -52,11 +52,11 @@ void update_pointer_system(rdpContext* context, POINTER_SYSTEM_UPDATE* pointer_s
 
 		default:
 			WLog_ERR(TAG,  "Unknown system pointer type (0x%08X)", pointer_system->type);
-			break;
```

Comment:
I tried to preserve the old functionality if something isn't implemented warn but don't count it as error. If you think raising an error makes more sense I can change it.